config.json

```
@@ -1,11 +1,11 @@
 {
   "exchange": "binance",
-  "timeframe": "5m",
+  "timeframe": "30m",
   "max-open-trades": 3,
   "exposure-per-trade": 100,
   "starting-capital": 1000,
-  "backtesting-from": "2020-09-01",
-  "backtesting-to": "2021-09-01",
+  "backtesting-from": "2021-05-01",
+  "backtesting-to": "2021-11-01",
```

```suggestion
  "backtesting-from": "2021-01-01",
  "backtesting-to": "2021-07-01",
```

Maybe have it from the first of January? That looks a bit neater!

---

Indeed!